public/app/plugins/panel/xychart/models.gen.ts

```diff
@@ -1,85 +0,0 @@
-import {
-  OptionsWithTooltip,
-  OptionsWithLegend,
-  LineStyle,
-  VisibilityMode,
-  HideableFieldConfig,
-  AxisConfig,
-  AxisPlacement,
-  ColorDimensionConfig,
-  ScaleDimensionConfig,
-  TextDimensionConfig,
-} from '@grafana/schema';
-import {
-  DimensionSupplier,
-} from 'app/features/dimensions';
-
-import { DEFAULT_POINT_SIZE } from './config';
-
-// export enum ScatterLineMode {
-//   None = 'none',
-//   Linear = 'linear',
-//   Smooth
-//   r2, etc
-// }
-
-export enum ScatterShow {
-  Points = 'points',
-  Lines = 'lines',
-  PointsAndLines = 'points+lines',
-}
-
-export enum SeriesMapping {
-  Auto = 'auto',
-  Manual = 'manual',
-}
-
-export interface ScatterFieldConfig extends HideableFieldConfig, AxisConfig {
```

Comment:
this is an old/duplicate file that *almost* is the same as panelcfg.gen.ts 🤯 

Most of this PR is updating references to the generated one rather than this one